src/graphic/helper/roundSector.ts

```diff
@@ -165,11 +189,24 @@ export function buildPath(ctx: CanvasRenderingContext2D | PathProxy, shape: {
     }
     // is a circular or annular sector
     else {
+        let icrStart;
```

Comment:
Should we give them a default `0` vaue? Seems undefined initial value will lead to some `NaN` values, e.g. `ocrs`, `ocre`, `icrs`, `icre` seems will be `NaN`. I'm not sure if it will cause unexpected issues.

Reply:
I think it's unnecessary. I made this change in the commit 3bb9b1be6758ecda9503fc786927dd5077004684. All the conditions about numeric comparison won't pass as `NaN > e` is false.

https://github.com/ecomfe/zrender/blob/3bb9b1be6758ecda9503fc786927dd5077004684/src/graphic/helper/roundSector.ts#L225

https://github.com/ecomfe/zrender/blob/3bb9b1be6758ecda9503fc786927dd5077004684/src/graphic/helper/roundSector.ts#L254

https://github.com/ecomfe/zrender/blob/3bb9b1be6758ecda9503fc786927dd5077004684/src/graphic/helper/roundSector.ts#L288